R/reprex_inform.R

```diff
@@ -0,0 +1,22 @@
+reprex_quiet <- function() {
+  as.logical(Sys.getenv("REPREX_QUIET", unset = "NA"))
+}
+
+local_reprex_quiet <- function(reprex_quiet, env = parent.frame()) {
+  withr::local_envvar(c(REPREX_QUIET = reprex_quiet), .local_envir = env)
+}
+
+reprex_inform <- function(message) {
+  quiet <- reprex_quiet() %|% is_testing()
+  if (quiet) {
+    invisible()
+  } else {
+    inform(message, class = "reprex_message")
```

Comment:
I don't think there's any need to set a class here, FWIW.

Reply:
Agreed. This was more me using this task to re-familiarize myself with the classed condition mindset. But it still seems neutral to positive.